testdata/validate-data/data/value_type_violation.xml

```diff
@@ -14,75 +14,75 @@
         For the text-prop the encoding was switched
      -->
 
-    <resource label="Empty" restype=":ClassWithEverything" id="id_0"/>
+    <resource label="Empty" restype=":ClassWithEverything" id="empty_ok"/>
```

Comment:
**praise** I like the id strings which explain their intention, makes it so much easier to read.